crates/services/txpool_v2/src/storage/graph.rs

```diff
@@ -735,7 +735,9 @@ impl Storage for GraphStorage {
                     }
                 }
                 Input::Contract(Contract { contract_id, .. }) => {
-                    if !self.contracts_creators.contains_key(contract_id) {
+                    if utxo_validation
+                        && !self.contracts_creators.contains_key(contract_id)
+                    {
```

Comment:
We always want to require contract to exist=D Regardless of `utxo_validation`.

Otherwise executor will fail